project_euler/problem_038/sol1.py

```diff
@@ -0,0 +1,77 @@
+"""
+Project Euler Problem 38: https://projecteuler.net/problem=38
+
+Take the number 192 and multiply it by each of 1, 2, and 3:
+
+192 × 1 = 192
+192 × 2 = 384
+192 × 3 = 576
+
+By concatenating each product we get the 1 to 9 pandigital, 192384576. We will call
+192384576 the concatenated product of 192 and (1,2,3)
+
+The same can be achieved by starting with 9 and multiplying by 1, 2, 3, 4, and 5,
+giving the pandigital, 918273645, which is the concatenated product of 9 and
+(1,2,3,4,5).
+
+What is the largest 1 to 9 pandigital 9-digit number that can be formed as the
+concatenated product of an integer with (1,2, ... , n) where n > 1?
+"""
+
+
+def is_9_palindromic(n: int) -> bool:
+    """
+    Checks whether n is a 9-digit 1 to 9 pandigital number.
```

Comment:
Does it check whether `n` is pandigital or palindromic? Please update either the docstring or function name accordingly.

Reply:
Sorry, you're right - the function name is wrong.